Slingshot.Api/ng/Scripts/app.js

```diff
@@ -205,6 +205,15 @@ function IsSiteLocationParam(paramName) {
 
         function initialize() {
             $scope.formData.repositoryUrl = getQueryVariable("repository");
+            var extraParams = getQueryVariable("extras");
```

Comment:
Why use a single param for all the data rather than promote each to its own query string param? The JSON param makes it a be tricky to hand write in the address bar (e.g. for testing).

Is it to avoid potential conflicts with other existing query params?